integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/PostgreSQLContainer.java

```diff
@@ -0,0 +1,153 @@
+import org.rnorth.ducttape.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.Network;
+
+public class PostgreSQLContainer extends BaseContainer {
+  public static final Logger LOG = LoggerFactory.getLogger(PostgreSQLContainer.class);
+  public static final PGImageName DEFAULT_IMAGE = PGImageName.VERSION_13;
+  public static final String HOST_NAME = "gravitino-ci-pg";
+  public static final int PG_PORT = 5432;
+  public static final String USER_NAME = "root";
+  public static final String PASSWORD = "root";
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  protected PostgreSQLContainer(
+      String image,
+      String hostName,
+      Set<Integer> ports,
+      Map<String, String> extraHosts,
+      Map<String, String> filesToMount,
+      Map<String, String> envVars,
+      Optional<Network> network) {
+    super(image, hostName, ports, extraHosts, filesToMount, envVars, network);
+  }
+
+  @Override
+  protected void setupContainer() {
+    super.setupContainer();
+    withLogConsumer(new PrintingContainerLog(format("%-14s| ", "PostgreSQLContainer")));
+  }
+
+  @Override
+  public void start() {
+    super.start();
+    Preconditions.check("PostgreSQL container startup failed!", checkContainerStatus(5));
+  }
+
+  @Override
+  protected boolean checkContainerStatus(int retryLimit) {
+    int nRetry = 0;
+    boolean isPostgreSQLContainerReady = false;
+    int sleepTimeMillis = 20_00;
+    while (nRetry++ < retryLimit) {
+      try {
+        String[] commandAndArgs =
+            new String[] {
+              "pg_isready", "-h", "localhost", "-U", getUsername(),
+            };
+        Container.ExecResult execResult = executeInContainer(commandAndArgs);
+        if (execResult.getExitCode() != 0) {
+          String message =
+              format(
+                  "Command [%s] exited with %s",
+                  String.join(" ", commandAndArgs), execResult.getExitCode());
+          LOG.error("{}", message);
+          LOG.error("stderr: {}", execResult.getStderr());
+          LOG.error("stdout: {}", execResult.getStdout());
+        } else {
+          LOG.info("PostgreSQL container startup success!");
+          isPostgreSQLContainerReady = true;
+          break;
+        }
+        LOG.info(
+            "PostgreSQL container is not ready, recheck({}/{}) after {}ms",
+            nRetry,
+            retryLimit,
+            sleepTimeMillis);
+        await().atLeast(sleepTimeMillis, TimeUnit.MILLISECONDS);
+      } catch (RuntimeException e) {
+        LOG.error(e.getMessage(), e);
+      }
+    }
+
+    return isPostgreSQLContainerReady;
+  }
+
+  public void createDatabase(TestDatabaseName testDatabaseName) {
+    String pgJdbcUrl =
+        StringUtils.substring(
+                getJdbcUrl(testDatabaseName), 0, getJdbcUrl(testDatabaseName).lastIndexOf("/"))
+            + "/";
```

Comment:
Can be simplified with `getJdbcUrl("")`?

Reply:
I'll use `postgreSQLContainer.getJdbcUrl()` here.